openedx/core/djangoapps/discussions/tests/test_admin.py

```diff
@@ -0,0 +1,30 @@
+from django.test import TestCase
+from django.urls import reverse
+from django.contrib.auth.models import User
+from django.test import Client
+from openedx.core.djangoapps.discussions.models import DiscussionsConfiguration
+
+
+class DiscussionsConfigurationAdminTest(TestCase):
+    """
+    Tests for discussion config admin
+    """
+    def setUp(self):
+        super().setUp()
+        self.superuser = User.objects.create_superuser('admin', 'admin@example.com', 'password')
+        self.client = Client()
+        self.client.login(username='admin', password='password')
+
+        self.discussion_config = DiscussionsConfiguration.objects.create(
+            context_key='course-v1:MITx+Discussion.Test+06_25_2024',
+            provider_type='openedx',
+        )
```

Comment:
Might not be a good idea to add it as part of the test case, Most tests would want to create their own objects and in different ways so this can become useless as a common attr.